.eslintrc.js

```diff
@@ -78,5 +78,9 @@ module.exports = {
     'react/jsx-props-no-spreading': 'off',
     'react/jsx-one-expression-per-line': 'off',
     'prettier/prettier': 'off',
+    'no-underscore-dangle': 'off',
```

Comment:
Could you configure it like this : 
```suggestion
    'no-underscore-dangle': ["error", { "allow": ["_id"] }]
```